docs/docsite/rst/community/communication.rst

```diff
@@ -30,98 +30,51 @@ Request it in the `forum topic <https://forum.ansible.com/t/requesting-a-forum-g
 Real-time chat
 ==============
 
-For real-time interactions, conversations in the Ansible community happen over two chat protocols: Matrix (recommended) and IRC.
-The main Matrix and IRC channels exchange messages.
-This means you can choose whichever protocol you prefer for the main channels.
+For real-time interactions, conversations in the Ansible community happen over the Matrix protocol.
 
 .. note::
 
-  Although you can choose either Matrix or IRC, please take into account that many Ansible communities use only Matrix.
+  The :ref:`ansible_forum` is our default communication platform. Make sure you joined and use it before considering other options like Matrix.
 
-Join a channel any time to ask questions, participate in a Working Group meeting, or just say hello.
-
-Ansible community on Matrix
----------------------------
-
-To join the community using Matrix, you need two things:
+Join a channel any time to chat with fellow enthusiasts, participate in a Working Group meeting, or just say hello. To do it, you need:
 
 * a Matrix account (from `Matrix.org <https://app.element.io/#/register>`_ or any other Matrix homeserver)
 * a `Matrix client <https://matrix.org/clients/>`_ (we recommend `Element Webchat <https://app.element.io>`_)
 
 The Ansible community maintains its own Matrix homeserver at ``ansible.im``, however, public registration is currently unavailable.
 
-Matrix chat supports:
-
-* persistence (when you log on, you see all messages since you last logged off)
-* edits (Let you fix typos and so on. **NOTE** Each edit you make on Matrix re-sends the message to IRC. Please try to avoid multiple edits!)
-* replies to individual users
-* reactions/emojis
-* bridging to IRC
-* no line limits
-* images
-
 The room links in the :ref:`general_channels` or the :ref:`working_group_list` list will take you directly to the relevant rooms.
-
-If there is no appropriate room for your community, please create it.
+If there is no appropriate room for your community, please create it if needed.
```

Comment:
```suggestion
If there is no appropriate room for your community, please feel free to create one.
```

Reply:
@oraNod i wanted to make them think if they really need it. can we rephrase it somehow wrt the intent? There are a lot of ghost towns

Reply:
I'd suggest we don't mention the option to create a new matrix room at all. They should be active on the forum first and have a good set of participants before we create more  potentially 'ghost' rooms in matrix.

Reply:
I like the suggesting not to have it here, removed, thanks